core/models.py

```diff
@@ -254,3 +257,34 @@ class ScheduledEventRunInstance(BaseEntity):
     def __str__(self):
         return self.scheduled_event.name + ' - ' + self.status
 
+
+class FrozenData(BaseEntity):
+    """Stores table names for "frozen" data"""
+    status_choices = (
+        ('p', 'Pending'),
+        ('r', 'Running'),
+        ('c', 'Completed'),
+        ('e', 'Errored'),
+    )
+
+    parent_db_table = models.CharField(max_length=200, null=False)
+    frozen_db_table = models.CharField(max_length=200, null=False)
+    completed = models.CharField(max_length=1, choices=status_choices, default='p')
```

Comment:
May I rename this to `status`? Not sure what's happening in the background, so don't want to just change it ...